src/EA.Weee.RequestHandlers/AatfEvidence/Reports/GetEvidenceNoteReportHandler.cs

```diff
@@ -38,18 +38,22 @@ public async Task<CSVFileData> HandleAsync(GetEvidenceNoteReportRequest request)
                     request.RecipientOrganisationId,
                     request.AatfId,
                     request.TonnageToDisplay == TonnageToDisplayReportEnum.Net);
+
+            //Note: I am not sure what I am doing here -- this handler is likely to be shared 
```

Comment:
@GuidoWorkDev yeah put these columns back in. The request has a AatfId parameter... you can check that, so if !request.AatfId.HasValue then add the column.